### PR TITLE
Patch in ondrej ppa for Citation-API stack

### PR DESCRIPTION
**Pull request by baccenfutter**
# Changes

Patch in Ondreij PPA for PHP in Citation-API stack.

# CR

 - please update the stable PR post-merge

Related: easybib/ops#218

### CODE REVIEW

File: stack-citationapi/recipes/role-phpapp.rb [hunk 1]
```diff
@@ -1,17 +1,61 @@
...
-include_recipe 'php::module-zlib'
-include_recipe 'php::module-intl'
-include_recipe 'php::module-gearman'
-include_recipe 'php::module-zip'
-include_recipe 'php::module-zlib'
-include_recipe 'php::module-opcache'
-include_recipe 'php::module-tidy'
+php_version = node['stack_api']['php_version']
 
-include_recipe 'tideways'
```
Comment (baccenfutter) [inline stack-citationapi/recipes/role-phpapp.rb]: pulled in by `ies::role-phpapp`

File: stack-citationapi/recipes/role-phpapp.rb [hunk 2]
```diff
@@ -1,17 +1,61 @@
-include_recipe 'ies::role-generic'
-include_recipe 'nginx-app::server'
```
Comment (baccenfutter) [inline stack-citationapi/recipes/role-phpapp.rb]: pulled in by `ies::role-phpapp`

**Comment by till** [comment]
@baccenfutter if I merge this, can you rotate?

**Comment by baccenfutter** [comment]
@till, locally the tests all pass. Not sure what's up with Travis? They are not even displaying the console. It is loading forever. Triggered a rebuild twice and double checked locally twice.

I guess we'll have to wait until Travis resolves their issues:
- https://statusgator.com/services/travis-ci

**Comment by baccenfutter** [comment]
Seems as if Travis finally completed the build since I last looked into the issue.

@till could you please review and merge?

**Pull request merged**